### PR TITLE
The behavior of clickhouse-keeper to create znode is consistent with zookeeper

### PR DESCRIPTION
**Pull request by nicelulu**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Improvement

Changelog entry:
Improved the existence condition judgment and empty string node judgment when clickhouse-keeper creates znode.

### CODE REVIEW

**Comment by nicelulu** [comment]
```
2021.08.03 15:25:57.116413 [ 5657 ] {2f0fdf4a-1326-482a-8e84-87c4dca1794c} <Error> executeQuery: Code: 999. Coordination::Exception: Bad arguments, path: /clickhouse/. (KEEPER_EXCEPTION) (version 21.9.1.7648) (from [::1]:48922) (comment: '/usr/share/clickhouse-test/queries/0_stateless/01034_move_partition_from_table_zookeeper.sh') (in query: CREATE TABLE src (p UInt64, k String, d UInt64) ENGINE = ReplicatedMergeTree('/clickhouse//src3', '1') PARTITION BY p ORDER BY (p, k, d);), Stack trace (when copying this message, always include the lines below):

```
```
ReplicatedMergeTree('/clickhouse//src3', '1')
```
Maybe it is creating /clickhouse/, which will also cause the same error under zookeeper: Bad arguments

**Comment by nicelulu** [comment]
ZooKeeper has the same error.
```
CREATE TABLE src
(
    `p` UInt64,
    `k` String,
    `d` UInt64
)
ENGINE = ReplicatedMergeTree('/clickhouse//src3', '1')
PARTITION BY p
ORDER BY (p, k, d)

Query id: f2f0eb98-2848-44ae-9255-7c53371b993a


Received exception from server (version 21.3.13):
Code: 999. DB::Exception: Received from 10.10.10.10:9000. DB::Exception: Bad arguments, path: /clickhouse/.

0 rows in set. Elapsed: 0.007 sec.
```

File: tests/integration/test_keeper_back_to_back/test.py [hunk 1]
```diff
@@ -90,6 +90,45 @@ def test_sequential_nodes(started_cluster):
...
+            genuine_zk.create("/test_sequential_nodes_1/a", sequence=True)
+        except Exception as ex:
+            genuine_throw = True
+
+        try:
+            fake_zk.create("/test_sequential_nodes_1/a", sequence=True)
+        except Exception as ex:
+            fake_throw = True
+
+        assert genuine_throw == fake_throw
```
Comment (tavplubix) [inline tests/integration/test_keeper_back_to_back/test.py:115]: It would be better to check that both `genuine_throw` and `fake_throw` are `True`

**Review by tavplubix — APPROVED** [review]

File: tests/integration/test_keeper_back_to_back/test.py (hunk 1 above)
Comment (nicelulu) [reply tests/integration/test_keeper_back_to_back/test.py:116]: Thanks.

**Comment by alesapin** [comment]
@mergifyio update

**Comment by alesapin** [comment]
@Mergifyio update

**Review by alesapin — APPROVED** [review]
Nice catch, LGTM

**Comment by alesapin** [comment]
Unrelated failures in tests.

**Pull request merged**